hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/JavaTypesArrayTest.java

```diff
@@ -283,7 +403,12 @@ private static class Basic {
 		@Id
 		@GeneratedValue
 		Integer id;
-		String[] stringArray;
+		String[] stringArrayNoAnnotations;
+		@Array(length = 10) // NOTE that the property `length` is required
```

Comment:
Please, avoid this kind of comments. A compilation error is more then enough.

The reason I've asked you to test what happened when the length is not specified is because I didn't notice there was no default value. You could have just told me.

Reply:
that was a note to myself... it's gone